Content.Shared/Nutrition/Components/SmokableComponent.cs

```diff
@@ -13,16 +13,16 @@ public sealed partial class SmokableComponent : Component
         /// <summary>
         ///     Solution inhale amount per second.
         /// </summary>
-        [DataField("inhaleAmount")]
+        [DataField("inhaleAmount"), ViewVariables(VVAccess.ReadWrite)]
         public FixedPoint2 InhaleAmount { get; private set; } = FixedPoint2.New(0.05f);
 
-        [DataField("state")]
+        [DataField("state"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
dont really think it's necessary for sprites

Reply:
also it doesnt update appearance so it wouldnt do anything